components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -2222,7 +2222,7 @@ document.observe('xwiki:dom:loaded', function()
   </object>
   <object>
     <name>PhenoTips.PatientSheetCode</name>
-    <number>8</number>
+    <number>1</number>
```

Comment:
I do and it seems to work. However moving the whole object up should be done as well, I somehow did not think of that (since the slowest code is in this object anyhow). For the record, all stuff related to UI elements visible 90% of the time on the first page should be moved up, like the date-picker widget (so that date input fields don't pop up out of nowhere(or at least do it faster),like you commented for another commit)